packages/shared/src/utils/platform/platform.js

```diff
@@ -12,18 +12,21 @@ export const isMT5 = () =>
     /^\/mt5/.test(window.location.pathname) ||
     (/^\/(br_)/.test(window.location.pathname) && window.location.pathname.split('/')[2] === 'mt5');
 
-export const getPlatformHeader = routing_history => {
-    if (isBot() || isNavigationFromPlatform(routing_history, routes.bot)) return 'DBot';
-    if (isMT5() || isNavigationFromPlatform(routing_history, routes.mt5)) return 'DMT5';
-    if (isNavigationFromPlatform(routing_history, routes.smarttrader)) return 'SmartTrader';
-    return 'DTrader';
-};
+export const isTrader = () =>
+    (/^\//.test(window.location.pathname) && window.location.pathname.length === 1) ||
+    (/^\/(br_)/.test(window.location.pathname) && window.location.pathname.split('/')[2] === '');
+
+export const getPlatformIconOrHeader = (routing_history, isHeader) => {
```

Comment:
- Please use constants for platform names.
- Perhaps we could refactor this to either return an object with information regarding a platform (rather than using a single function that can return two different things) or by writing two separate functions, e.g.:

```JS
export const getPlatformInformation = (routing_history) => {
    let header, icon;
    // Logic to determine Header + Icon
    
    return {
        header,
        icon,
    }
}
```

or


```JS
export const getPlatformHeader = () => {
        // Logic.
}
```

and

```JS
export const getPlatformIcon = () => {
        // Logic.
}
```

Reply:
For issue #1: I saw in the other card Carol used constants for platform name, so I did not do that, I believe in merge it will be changed.
For issue #2: I did that because both of those functions had the same code, and I wanted to remove the duplication, so increase the code maintainability. But in both of your suggested solutions I should duplicate the code again.
Isn't it better to make constants for Icon names as well as platform and return those constants instead?

Reply:
- #1: Sure, perhaps we could add a `TODO` so we won't forget.
- #2: I'm trying to keep functions clear and understandable, so when they have a single responsibility and expected outcome it'll be easier for the next developer to know what's happening. If you want to avoid duplicating logic, the first example (returning an object) will avoid that.